app/javascript/components/stylings/Spinner.jsx

```diff
@@ -1,15 +1,14 @@
 import React from 'react';
 import { Spinner as BootstrapSpinner } from 'react-bootstrap';
 
-export default function Spinner(props) {
+export default function Spinner() {
   return (
     <BootstrapSpinner
       as="span"
       animation="grow"
       size="sm"
       role="status"
       aria-hidden="true"
-      {...props}
```

Comment:
This will introduce a bug in the `UI` once we need to overwrite the default attributes.